x-pack/docs/en/security/es-security-principles.asciidoc

```diff
@@ -0,0 +1,55 @@
+[[es-security-principles]]
+== {es} security principles
+Protecting your {es} cluster and the data it contains is of utmost importance.
+There are numerous security practices and considerations for keeping your
+cluster safe, such as not saving passwords in plain text files or making sure
```

Comment:
@bytebilly, I'm good with adding the actions that users _should_ do instead of what not to do. I tried to reframe each of the section titles around the user action such as **Run Elasticsearch with security enabled** instead of **Do not run Elasticsearch without security**, which is also more positive. I think we can extend that mantra to this additional section as well. What are the principles that you want to add? Are you thinking of something like, **Use secure settings whenever possible**?

Reply:
Yes, we could also link the secure settings documentation. I like the idea to use positive messages for all sections!

Reply:
The first paragraph feels little flowery compared to the fairly direct "don't put it on the internet, even with security enabled" kinds of advice you give below.

Reply:
@nik9000, I think you're right that stronger language is appropriate here. I'm thinking something like:
>Protecting your {es} cluster and the data it contains is of utmost importance. The following principles are foundational to running {es} in a secure manner and should be integrated into every production deployment of {es}.

Reply:
Sounds good to me.